### PR TITLE
chore(eslint): fix eslint,  vscode auto save

### PR DESCRIPTION
**Pull request by davidholyko**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,17 +1,35 @@
 {
   "env": {
     "browser": true,
-    "es2021": true,
-    "node": true
+    "es2020": true,
```
Comment (davidholyko) [inline .eslintrc.json:4]: es2021 is broken.

https://github.com/NullVoxPopuli/eslint-plugin-decorator-position/issues/138

File: .eslintrc.json [hunk 2]
```diff
@@ -1,17 +1,35 @@
...
     "ecmaFeatures": {
       "jsx": true
     },
-    "ecmaVersion": 12,
+    "ecmaVersion": 11,
     "sourceType": "module"
   },
   "plugins": ["react"],
-  "rules": {}
+  "rules": {
```
Comment (davidholyko) [inline .eslintrc.json:17]: feel free to add more rules to this branch or suggest as a comment. @trybick

File: .vscode/settings.json [hunk 3]
```diff
@@ -5,5 +5,11 @@
     "titleBar.activeBackground": "#07484C",
     "titleBar.activeForeground": "#EEFCFD"
   },
-  "editor.wordWrapColumn": 160
+  "editor.wordWrapColumn": 160,
```
Comment (davidholyko) [inline .vscode/settings.json:8]: enables extra long comments to not be wrapped (edge case)

File: .vscode/settings.json [hunk 4]
```diff
@@ -5,5 +5,11 @@
...
+  "[javascript]": {
```
Comment (davidholyko) [inline .vscode/settings.json:9]: enables auto save with prettier that VSCode broke

**Review by trybick — APPROVED** [review]
LGTM. Two things, which we can clean up before or after this goes in 

- In D3Chart getting an eslint error saying we need prop types
- I would suggest adding  `"prettier/prettier": 1,"` to the eslint file so we can warnings for incorrect formatting. But it won't work unless we install the prettier eslint extension. I can set that up after this goes in if you think that sounds good.

**Pull request merged**